CNAME

```diff
@@ -1 +0,0 @@
-wo-ist-markt.de
```

Comment:
Please state in the commit message **why** you remove the file. As mentioned tonight it is not used since a while. It would also be nice if you could link the relevant commit hash.